charts/yunikorn-history-server/templates/deployment.yaml

```diff
@@ -39,6 +39,7 @@ spec:
         - name: "yunikorn-history-server"
           image: "{{ include "yunikorn-history-server.image" . }}"
           imagePullPolicy: "{{ $imagePullPolicy }}"
+          command: ["/app/yunikorn-history-server"]
```

Comment:
We already have this command to the `Entrypoint` of our docker image. Therefore, we do not need this.
```suggestion
```

Reply:
Agree.
I will remove this in next commit.